jpa/subsystem/src/main/java/org/jboss/as/jpa/messages/JpaLogger.java

```diff
@@ -752,4 +754,8 @@
     DeploymentUnitProcessingException differentSearchModuleDependencies(String deployment, String searchModuleName1, String searchModuleName2);
 
     // id = 72, value = "Could not obtain TransactionListenerRegistry from transaction manager")
+
+    @Message(id = 73, value = "Invalid class format for deployment class %s.")
```

Comment:
A message like "Transformation of class %s failed" is probably more appropriate.  The details will be included in the nested exception.

Reply:
easy change, thanks